### PR TITLE
Ignore purging read only tables

### PR DESCRIPTION
**Pull request by zecho**
This is a hotfix to allow ignoring entities marked as read only like views etc. It also use properly the tables schema. Tested with PostgreSQL.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-    "name": "hautelook/alice-bundle",
-    "description": "Symfony2 Bundle to manage fixtures with Alice and Faker.",
+    "name": "zecho/alice-bundle",
```
Comment (theofidry) [inline composer.json]: this line should be reverted

Comment (zecho) [inline composer.json]: Sorry, added to do local checks and committed it by mistake.

**Comment by grigoryosifov** [comment]
+1

**Comment by theofidry** [comment]
Coming back on that one:

- I feel very uncomfortable to merge anything on 1.x: the tests are broken/outdated and hard to fix due to a lot of infrastructure complexity (which has been fixed with 2.x)
- The case itself: I'm not sure why you don't want the read tables should not be deleted/truncated either
- Related tot he case above: if this is something generic and the reason is valid, I think this should be fixed in doctrine/data-fixtures instead where the purgers live

**Comment by theofidry** [comment]
Closing this one as I don't think this is going to happen. I'll re-open if there is a game changer argument though :)

**Pull request closed without merging**